jcommune-view/jcommune-web-view/src/main/webapp/WEB-INF/web.xml

```diff
@@ -151,6 +151,16 @@
         <url-pattern>/*</url-pattern>
     </filter-mapping>
 
+    <!--If request method to login page isn't GET or POST, filter changes it to GET-->
```

Comment:
It is needed to avoid getting http status `405-Method not allowed`. According to security-context.xml only authenticated users can have access to `/api/spam-rules`, so when Anonymous user performs [PUT] request to `/api/spam-rules` it is redirected to `/login` but there is no mapping specified to serve [PUT] request and we recieve response with status 405. Last time we discussed about this case and you suggested to add another `<http>` tag for [PUT] request in security-context.xml but I didn't find solution. If my solution is unacceptable, can you give me a clue?